content/docs/1.7.0/advanced-resources/backing-image/backing-image-encryption.md

```diff
@@ -42,6 +42,13 @@ spec:
 > - `backing-image`: Specify the name of the backing image to be cloned.
 > - `encryption`: Set the value to `ignore` to directly clone the backing image. If the value is not given, Longhorn use `ignore` as default value.
 
+Longhorn also supports cloning the backing image via UI.
+1. Go to `Setting -> Backing Image` page and click `Create Backing Image` button.
+2. Select `Created From` to `Clone From Existing Backing Image` and select `Encryption` to `ignore`.
+3. Click `OK`
```

Comment:
```suggestion
You can also clone a backing image using the Longhorn UI.

1. Go to **Setting** > **Backing Image**.

1. Click **Create Backing Image**.

1. Configure the following settings:

  - **Created From**: Select **Clone From Existing Backing Image**.
  - **Encryption**: Select **Do not encrypt**.

1. Click **OK**.
```

Reply:
Given that the other values are `Encrypt` and `Decrypt`, `Do not encrypt`  is more appropriate than `ignore`.

Reply:
But `ignore` is the common word in longhorn settings to inherit from global settings or as default option. You can see lots of settings having ignore in their options. 

I won't update the wording at this moment, but we can review the option wordings in [LEP](https://github.com/longhorn/longhorn/blob/master/enhancements/20240612-backing-image-encryption-and-clone-supprt.md#backing-image-data-source) reivew stage.

Reply:
`encrypt`, `decrypt`, `ignore` is the extra operation when doing cloning 
this is not only shown on the UI, but will be used in the system as well
So this needs to be a single and simple word.

you can check our settings, we use lots of `ignore` as default value, ref: https://longhorn.io/docs/1.6.2/references/settings/
maybe in the future we can come up with a more appropriate and common word for this case in all the settings.